generators/app/tasks/installDependencies.js

```diff
@@ -15,7 +15,14 @@ const DEPENDENCIES = [
   'react-router',
   'react-router-dom',
   'react-spinkit',
-  'wolox-equalizer'
+  'wolox-equalizer',
+  'babel-core',
+  'babel-runtime',
+  'eslint-import-resolver-babel-module',
```

Comment:
this is both in dev and not dev dependencies. Right now. It should only be in devDependencies